build-patched-kao-kcmo-images.sh

```diff
@@ -72,14 +72,15 @@ function patch_and_push_image() {
     rhpkg clone containers/crc-${image_name}
     pushd crc-${image_name}
     git remote add upstream git://pkgs.devel.redhat.com/containers/ose-${image_name}
-    # Just fetch the upstream/rhaos-4.11-rhel-8 instead of all the branches and tags from upstream
-    git fetch upstream rhaos-4.11-rhel-8 --no-tags
-    git checkout --track origin/rhaos-4.11-rhel-8
+    # Just fetch the upstream/rhaos-4.12-rhel-8 instead of all the branches and tags from upstream
+    git fetch upstream rhaos-4.12-rhel-8 --no-tags
+    git checkout --track origin/private-rhaos-4.12-rhel-8
```

Comment:
Would be good to get this from `$version`, or to have a constant whose value is `4.12`, we would not need to change many places for each new openshift version.

Reply:
Yes, constant would be much better instead extract it from `$version` .